src/routes/transactions/mappers/common/safe-app-info.mapper.spec.ts

```diff
@@ -10,12 +10,15 @@ describe('SafeAppInfo mapper (Unit)', () => {
     getSafeApps: jest.fn(),
   } as unknown as SafeAppsRepository);
 
+  let mapper: SafeAppInfoMapper;
+
   beforeEach(async () => {
     jest.clearAllMocks();
+    mapper = new SafeAppInfoMapper(safeAppsRepositoryMock);
   });
 
   it('should get a null SafeAppInfo for a transaction with no origin', async () => {
-    const mapper = new SafeAppInfoMapper(safeAppsRepositoryMock);
+    mapper = new SafeAppInfoMapper(safeAppsRepositoryMock);
```

Comment:
You don't need this line. You can assume that you have a `mapper` at this stage (due to `beforeEach`)

Reply:
Oh yes, sorry these lines weren't removed by mistake, I should have seen this in the review 🤦 Thanks! 🙂 Fixed in [a9d7769](https://github.com/5afe/safe-client-gateway-nest/pull/236/commits/a9d776911ee96604a21921ca6f63720492e3dee2)